pyccl/massfunction.py

```diff
@@ -0,0 +1,72 @@
+from . import halos as hal
+from .pyutils import deprecated
+from .errors import CCLError
+from .power import sigmaM  # noqa
+
+
+@deprecated(hal.MassFunc)
+def massfunc(cosmo, halo_mass, a, overdensity=200):
+    """Halo mass function, dn/dlog10M.
+    Args:
+        cosmo (:obj:`Cosmology`): Cosmological parameters.
+        halo_mass (float or array_like): Halo masses; Msun.
+        a (float): scale factor.
+        overdensity (float): overdensity parameter (default: 200)
+    Returns:
+        float or array_like: Halo mass function; dn/dlog10M.
```

Comment:
Doc string needs spaces between the parts.

Reply:
These were the previous docstrings. Sigh. Done!

Reply:
Ugh sorry! Never to late to make things better!